### PR TITLE
Used modal for view image on edit journey

### LINKED ISSUE #164: Full view the journey image in modal
## Overview
Right now we can do a full image view on blank and zoom in and out on blank not in current page.

## Conditions of Satisfaction
- View a full image on modal on current page
- Zoom in and zoom out the images of Journey in the list and Edit journey

## Required Resources
https://stackoverflow.com/questions/18432394/how-to-make-twitter-bootstrap-modal-full-screen
https://stackoverflow.com/questions/39858998/zoom-in-and-out-on-mouse-click-with-css

### PR DESCRIPTION
**Pull request by marobo**
Closes #164 

## Description of the Problem / Feature
Description on issue #164 

## Explanation of the solution
-  Shows up the image on modal, when the user click the journey image on edit journey

### CODE REVIEW

File: map/templates/map/viajen_form.html [hunk 1]
```diff
@@ -96,10 +96,21 @@ <h3 class="form-title">
         {% for photo in object.photos.all %}
             <div class="col-12">
                 <figure>
-                    <img class="edit-img-display" width="200" src="{{ photo.image.url }}">
+                    <img class="edit-img-display" width="200" title="{% trans 'View this image' %}" data-toggle="modal" data-target="#photo-{{ photo.id }}" src="{{ photo.image.url }}">
                     <a class="top-right" title="{% trans 'Delete this image' %}" href="{% url 'delete_photo' pk=photo.pk %}?next={{ request.path }}">X</a>
                 </figure>
             </div>
+            <!-- Modal -->
+            <div class="modal fade" id="photo-{{ photo.id }}" tabindex="-1" role="dialog" aria-labelledby="myModalLabel" aria-hidden="true">
```
Comment (eraulo) [inline map/templates/map/viajen_form.html:104]: @marobo, why are you using modal inside in the add journey?

Comment (marobo) [reply map/templates/map/viajen_form.html:104]: Hi @eraulo , That not inside add journey, but that for edit journey if there has any image!
Trying inspect the add journey page, I think you will never see the modal in there.

File: map/templates/map/viajen_form.html [hunk 2]
```diff
@@ -96,10 +96,21 @@ <h3 class="form-title">
...
+                <div class="modal-dialog modal-img">
+                    <div class="modal-content modal-img-content">
+                        <div class="modal-body modal-img-body">
+                            <img src='{{ photo.image.url }}' class="gallery-img" width="100%">
+                            <a class="close top-right" data-dismiss="modal" aria-hidden="true">X</a>
```
Comment (eraulo) [inline map/templates/map/viajen_form.html:109]:
```suggestion
                            <a class="close top-right" data-dismiss="modal" aria-hidden="true">&#119;</a>
```

File: map/templates/map/viajen_form.html (hunk 1 above)
Comment (eraulo) [reply map/templates/map/viajen_form.html:104]: So why are using inside in the edit journey?

Comment (marobo) [reply map/templates/map/viajen_form.html:104]: I have no comments

Comment (eraulo) [reply map/templates/map/viajen_form.html:104]: Why you don't have comment?

Comment (marobo) [reply map/templates/map/viajen_form.html:104]: Just the title of this pull request can be answer your question for why, why, & why... 
Diskulpa hau nia inglesh mak komplikadu oituan dalaruma halo Ano ladun komprende,
loos k lae @MarioAP?  :smile_cat:

Comment (eraulo) [reply map/templates/map/viajen_form.html:104]: Okey, ita nia comment kapas loos maibe la tuir tiha ona ita hotu nia hakarak. Nia susar iha ne'e!

Comment (eraulo) [reply map/templates/map/viajen_form.html:104]: Maibe hau merge deit ona ba futuro mak iha hadia diak liu tan.

**Pull request merged**